commands-yml/commands/mobile-command.yml

```diff
@@ -62,7 +62,7 @@
       | mobile:scrollBackTo | Scroll from one element to another | `{elementId, elementToId}` | `{elementId: 2, elementToId: 1}` |
       | mobile:viewportScreenshot | Like [screenshot](/commands/session/screenshot/) but only includes contents of viewport | <none> | <none> |
       | mobile:deepLink | Opens a deep-link URL for testing [Instant Apps](https://support.google.com/googleplay/answer/7240211?hl=en) | `{url, package}` | `{url: "https://www.site.com/", package: "com.site.SomeAndroidPackage"}` |
-
+      | mobile:mobileGetDeviceInfo | Gets device information like manufacturer and model. Read [GetDeviceInfo](https://github.com/appium/appium-uiautomator2-server/blob/master/app/src/main/java/io/appium/uiautomator2/handler/GetDeviceInfo.java) for more details. | <none> | <none> |
```

Comment:
should this be marked as Android-only?

Reply:
Yes, but this section is already `UiAutomator2 only` one => https://github.com/appium/appium/pull/11371/files#diff-29516863f13863f4f028f30051fe44a2R59

http://appium.io/docs/en/commands/mobile-command/